src/api_routes.rs

```diff
@@ -369,6 +370,7 @@ pub fn config(cfg: &mut ServiceConfig, rate_limit: &RateLimitCell) {
               .route("/community", post().to(user_block_community))
               .route("/instance", post().to(user_block_instance)),
           )
+          .route("/note/person", post().to(user_note_person))
```

Comment:
Putting it under `/person/note` instead of /account makes more sense to me, after all the note is also returned as part of read_person, and not anywhere with account data.